nowplaying/misc/saemubox.py

```diff
@@ -78,9 +77,10 @@ def __update(self):
 
             ids = data.split()  # several saemubox ids might come in one packet
             if ids:
-                if ids[-1] in self.valid_ids:  # only take last id
+                id = ids[-1].decode("utf-8")  # only take last id
```

Comment:
There is some test code in the end of this file that helped me python-3-ifiy this.